package.json

```diff
@@ -162,5 +152,12 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
+  },
+  "peerDependencies": {
+    "@emotion/core": "^10.0.20",
+    "@emotion/styled": "^10.0.17",
+    "emotion-theming": "^10.0.19",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2"
   }
```

Comment:
Peer dependencies should always be a range of supported versions so that the consuming application can pick the exact version and upgrade at its own pace: https://nodejs.org/en/blog/npm/peer-dependencies/. It's good to be lenient and only specify the minimum version:

```json
  "peerDependencies": {
    "@emotion/core": ">=10.0.0",
    "@emotion/styled": ">=10.0.0",
    "emotion-theming": ">=10.0.0",
    "react": ">=16.8.0",
    "react-dom": ">=16.8.0"
  }
```